src/Document/GalleryManager.php

```diff
@@ -34,7 +35,7 @@ public function update(GalleryInterface $gallery): void
         parent::save($gallery);
     }
 
-    public function getPager(array $criteria, $page, $limit = 10, array $sort = []): void
+    public function getPager(array $criteria, int $page, int $limit = 10, array $sort = []): PagerInterface
     {
         throw new \RuntimeException('Not Implemented yet');
```

Comment:
I know this change is not part of your PR, but just in case you can:
```suggestion
        throw new \BadMethodCallException('Not Implemented yet.');
```

Reply:
In Monday I will open other PR for it. After that I will merge it.

Reply:
#1934.